lib/tasks/import.rake

```diff
@@ -62,7 +62,7 @@ namespace :csv_load do
 
   task transactions: :environment do
     CSV.foreach('./db/data/transactions.csv', headers: true) do |row|
-      if row[4] == 'failure'
+      if row[4] == 'failed'
```

Comment:
Nice catch!

Reply:
This is after I managed to populate half the transactions with null, spelling is hard.